pontoon/batch/actions.py

```diff
@@ -69,6 +78,10 @@ def approve_translations(form, user, translations, locale):
         locale,
     )
 
+    before_level = _get_badge_level(
+        settings.BADGES_REVIEW_THRESHOLDS, user.badges_review_count
+    )
```

Comment:
Could we instead call this as something along the lines of `user.badges_review_level`? Where `badges_review_level` would be a user property with the logic similar to `_get_badge_level()`?